src/components/Form/fields.jsx

```diff
@@ -1,27 +1,22 @@
 import React from 'react'
-import Input from 'cozy-ui/transpiled/react/Input'
-import Textarea from 'cozy-ui/transpiled/react/Textarea'
+import TextField from 'cozy-ui/transpiled/react/MuiCozyTheme/TextField'
 
-function adaptField(WrappedComponent) {
-  class AdaptField extends React.Component {
-    render() {
-      const {
-        input,
-        meta: { valid },
-        ...rest
-      } = this.props
-      return <WrappedComponent {...input} {...rest} data-valid={valid} />
-    }
-  }
-  AdaptField.displayName = `adaptField(${getDisplayName(WrappedComponent)})`
-  return AdaptField
-}
-
-function getDisplayName(WrappedComponent) {
-  return WrappedComponent.displayName || WrappedComponent.name || 'Component'
-}
-
-const AdaptedInput = adaptField(Input)
-const AdaptedTextarea = adaptField(Textarea)
-
-export { AdaptedInput, AdaptedTextarea }
+export const DumbField = ({
+  input,
+  type,
+  labelProps,
+  isMultiline,
+  ...rest
+}) => (
+  <TextField
+    {...input}
+    {...rest}
+    type={type}
+    variant="outlined"
+    fullWidth={true}
+    multiline={isMultiline}
+    rows="2"
+    InputLabelProps={labelProps}
+    aria-label="dumbField"
```

Comment:
Mmh that seems wrong. Shouldn't we use the real label here?

Reply:
Le vrai label est utilisé pour créer un <label> alors qu'ici le aria-label ne sert que pour les tests afin de cibler l'<input> et non le <label>, afin de vérifier qu'il y a bien les input... or un <Textfield> peut devenir un input ou un textarea, distinction qui n'a pas d'importance dans les tests.

Reply:
Mmh non, [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) est utilisé par les technos d'aide à l'accessibilité, c'est pas juste pour les tests :). Et vu la doc je pense pas qu'on soit dans un bon cas d'utilisation.

Si `TextField` affiche un label (j'espere que c'est le cas), on devrait pouvoir utiliser `getByLabel` dans le test.

Reply:
Ok je vais changer je m'étais appuyé sur https://testing-library.com/docs/react-testing-library/cheatsheet `ByLabelText find by label or aria-label text content`, j'ai foncé tête baissée en voyant `aria-label`